.github/CONTRIBUTING.md

```diff
@@ -67,6 +67,16 @@ It is possible we already fixed the bug you're experiencing.
 
  - [ ] **Include a use case description**: In addition to describing the behavior of the feature you'd like to see added, it's helpful to also lay out the reason why the feature would be important and how it would benefit the wider Terraform ecosystem. Use case in context of 1 provider is good, wider context of more providers is better.
 
+### Documentation Contributions
+
+ - [ ] **Search for possible duplicate suggestions**: It's helpful to keep
+   suggestions consolidated to one thread, so do a quick search on existing
+	@@ -81,6 +81,8 @@ type.
+   "how do servers' schemas get merged?" in mind. This helps us ensure the
+   documentation is targeted, specific, and framed in a useful way.
```

Comment:
Copy-pasta 🍝  -- this suggestion takes the content from terraform-plugin-mux and changes the question to a pertinent one for this project

```suggestion
 - [ ] **Search for possible duplicate suggestions**: It's helpful to keep
   suggestions consolidated to one thread, so do a quick search on existing
   issues to check if anybody else has suggested the same thing. You can scope
   searches by the label `documentation` to help narrow things down.

 - [ ] **Describe the questions you're hoping the documentation will answer**:
   It's very helpful when writing documentation to have specific questions like
   "how do I implement a default value?" in mind. This helps us ensure the
   documentation is targeted, specific, and framed in a useful way.
```